docs/_tutorials/cifar-10.md

```diff
@@ -122,7 +122,7 @@ def initialize(args,
                collate_fn=None):
 ```
 
-For CIFAR-10 model, we initialize DeepSpeed its model (net) is created as below, to pass the raw `model`, `optimizer`, `args`, `parametersnd` and `trainset`.
+Here we we initialize DeepSpeed with CIFAR-10 model (`net`), `args`, `parameters` and `trainset`:
```

Comment:
Loving the edits, thanks for the help here. I think you accidently added an extra 'we' here.

Reply:
oops, thank you for catching that - fixed.